### DISCUSSION TITLE
Please vote

### DISCUSSION THREAD

**Discussion by tfius** [fairDataSociety/Fairdrive-DAO#1] [Proposals]
https://xdai.colony.io/colony/fairdrive/decisions/tx/0x64b6b3fed620733151b078afabd2661cc1822d7c47cd9a999e907c32a87d1b25

**Comment by mmohorko** [comment]
I am up for it, let's just quickly think about this: FDS and Fairdrive are 2 different entities. Fairdrive is soon going to be incorporated. How do we want to present the relationship between them in the future?
